### DISCUSSION TITLE
WBTC (ERC-20) to WBTC (SOL) stuck on "Awaiting completion on Solana..."

### DISCUSSION THREAD

**Discussion by ralopibralop** [wormhole-foundation/wormhole#333] [Problems / bugs] [answered]
Hi guys, was testing wormhole (before trying with a larger sum) and tried to move 0.0001 WBTC from ERC-20 to SOL. The process got stuck on "Awaiting completion on Solana..." for several hours already.

ERC-20 transactions:
https://etherscan.io/tx/0x89ab0001d93bbe58434ed2641f5ce33758871fc4883422cd3f16959799bbeea9
https://etherscan.io/tx/0x1f20ed88bbe663972d2e8522d3fc6093ee8c3cb6e420eaf56e61033e07da9711

Would appreciate help, thanks!

<img width="376" alt="Снимок экрана 2021-08-21 160838" src="https://user-images.githubusercontent.com/89306001/130325307-a1a17cdc-d5d3-4668-9bdf-3820e4ab4945.png">

**Comment by leoluk** [comment, marked as answer]
Looks like you got bit by a bug in wormholebridge.com - the destination token account wasn't created, but the lockup was. Wormhole itself correctly processed the transfer, but the tx failed on Solana since the account specified as the destination does not exist: https://explorer.solana.com/address/5PSFisN1TxJ6hih4DDUhCtfwLNUtTvCK8xLbtjDQtX4m

You need to reach out to the wormholebridge.com team so they can help you create the missing token account, then we can resubmit the VAA: https://github.com/solana-labs/oyster/issues

(`412e148131f84affa4c74438df5a1db7149124ad31dc22c4a14ff3b490aeb2fa` decodes to `5PSFisN1TxJ6hih4DDUhCtfwLNUtTvCK8xLbtjDQtX4m`).

---

Digest: dacba2e1e474824e9fe7bfd448720f5d255d14ae2e5965e1430fd9262b50a707

Signed VAA:

> 01000000011000cda7b1adc99caf9e40218a705ee9b6f04a92461a7a92cc716d31f1254fac2aac0927929fdf8f7a9d15cecc0379b580c135bb0ae54a75795aaad6b8677b0981f40101f93529dc696d116b3a82dbc79609f8f1e24176a720c5a20514aaee48e9ce07e37f61059fce6aa3ab00f6ac6f02c8857ed137300a21dc6de0c12cab0f9f52de630002e324267f7268b4a968dec1942b00f1583b0e8bf43fbf66af4cce1733f0eed00549283041e30fa6c386538531464e5439d0c30c31086bbade1bc37c710a23f9fa0104675824204b685d683ce44243d721da1478c5d84652866210a2334737579237bc56d5c27fb1db08459c50257f80830e92e3c922f3cc7c35139b0261644dbead6e01058f25eff5e1be2764c27b6de7f8c24781eff4e7860e41c61e68e068d3ab753791583474829e88d7b933bf9d2831852d6051451a4d4f423026bdc89b281a9fe3880006a5ccb7c6b71a71e408c08f329a63c6f65133588de414d45bde2a9cedc11ab0862f38d200d757ff1ca32c9c9bef38b3eb8d3bf188df823d8e32ef9eae1410f01e0107203374dc5b7494f617c0e37ee0fcbb6e3587b130084a5080dccc5949476896b05bf4ec976d307befde579d3797f65819ddccaef6ecdc7cad485777361d7b012a00081d6444421cda1a766a01c0b91d51ee41da1d2335a1bc4bdfa1bfdf4c69319c107a7d47dcb85de291118d2d8e676672fc9c8c65f659aeff0c6f1abe30c4ea404b0009c55b842c6011597ff7c1f8baabbb3bc7c3cbc92cce86843d06ab17be9e94d5be2f4e238f4cc791581e94e96fdabf5a2cfa9e96ed36d3366252019169c4fb31f8010afc578480f5d57ea26c7cb4a22af8e0e54183e3f2392876ec5a820f31a86ea0c52ba880e6fa17aa4906f124235323d853af35f2dce9742072d142b1e86d10e55d000be88110e8a528beb115034d536b63bcddbd08c049c131c836a4aac8ffb1b8bd55176ec738235f7dca5d13ac8af6bb08acae8c23260d909caa2593d3c899fca93e010d8576aa068b3ed5323bfba6b4994e7279c541339024f12dd43bdd4e4b429d74fc210eef1bcaaadfcfcfa23944742b8efdb5d656bb13791b801a80189de04dde51000f9528ceaee232c8e02a560eb29aed785a434398736b865dd07f6bc1352a3fdc61175c0f8a702c6cf5dcac059917b63b90edf716ad0d5e3166d68b87d235b4cfa801106d070630a8e0db9cc3655602faf354794e978ae1b7061cd6b2a88cc8159470fc64b4b0395cf1e8f68e6cdac658a755182c38c436dff429feb180fe8ffbeef8a60011129dad74f13de7cc8bb030acdd3fea9e8625695430388964ff2a187d5ebd5fcf0711192db14c19662bc2cd40eeb6d252d74c3ddbbd24244e19fb72645aae40250112c07249a872bf4603493f68a8c13fe3354c21c66faceff7affa3ea87865407cfb62bfe0d0c3a6c0fb8d2ca4cffd6e04d16ba6551664050774bcf1a18acf221aaf016120f28f100000001c0201000000000000000000000000b2eeea821e8eb68f1b461f7bb936ef9a3efce962412e148131f84affa4c74438df5a1db7149124ad31dc22c4a14ff3b490aeb2fa020000000000000000000000002260fac5e5542a773aa44fbcfedf7c193bc2c599080000000000000000000000000000000000000000000000000000000000002710

Decoded VAA:

```
  Nonce: (uint32) 28,
  SourceChain: (vaa.ChainID) ethereum,
  TargetChain: (vaa.ChainID) solana,
  SourceAddress: (vaa.Address) (len=32 cap=32) 000000000000000000000000b2eeea821e8eb68f1b461f7bb936ef9a3efce962,
  TargetAddress: (vaa.Address) (len=32 cap=32) 412e148131f84affa4c74438df5a1db7149124ad31dc22c4a14ff3b490aeb2fa,
  Asset: (*vaa.AssetMeta)(0xc0003f4090)({
   Chain: (vaa.ChainID) ethereum,
   Address: (vaa.Address) (len=32 cap=32) 0000000000000000000000002260fac5e5542a773aa44fbcfedf7c193bc2c599,
   Decimals: (uint8) 8
  }),
  Amount: (*big.Int)(0xc000729920)(10000)
```

**Comment by ralopibralop** [comment]
https://github.com/solana-labs/oyster/issues/196
